specs-go/v1/index.go

```diff
@@ -26,19 +26,19 @@ type Platform struct {
 	OS string `json:"os"`
 
 	// OSVersion is an optional field specifying the operating system
-	// version, for example `10.0.10586`.
+	// version, for example on Windows `10.0.10586`.
 	OSVersion string `json:"os.version,omitempty"`
 
 	// OSFeatures is an optional field specifying an array of strings,
 	// each listing a required OS feature (for example on Windows `win32k`).
 	OSFeatures []string `json:"os.features,omitempty"`
 
 	// Variant is an optional field specifying a variant of the CPU, for
-	// example `ppc64le` to specify a little-endian version of a PowerPC CPU.
+	// example `armv6l` to specify a particular CPU variant of the ARM CPU.
```

Comment:
I was unsure why `variant=ppc64le` is useful, because `architecture` would be always `ppc64le` in such a variant.
So I just thought standardization for ppc64le would be better deferred to post-v1.0.0, but not sure.